source/common/upstream/load_balancer_impl.cc

```diff
@@ -754,17 +755,36 @@ const HostVector& ZoneAwareLoadBalancerBase::hostSourceToHosts(HostsSource hosts
 EdfLoadBalancerBase::EdfLoadBalancerBase(
     const PrioritySet& priority_set, const PrioritySet* local_priority_set, ClusterStats& stats,
     Runtime::Loader& runtime, Random::RandomGenerator& random,
-    const envoy::config::cluster::v3::Cluster::CommonLbConfig& common_config)
+    const envoy::config::cluster::v3::Cluster::CommonLbConfig& common_config,
+    const absl::optional<envoy::config::cluster::v3::Cluster::SlowStartConfig> slow_start_config,
+    TimeSource& time_source)
     : ZoneAwareLoadBalancerBase(priority_set, local_priority_set, stats, runtime, random,
                                 common_config),
-      seed_(random_.random()) {
+      seed_(random_.random()),
+      slow_start_window_(slow_start_config.has_value()
+                             ? std::chrono::milliseconds(DurationUtil::durationToMilliseconds(
+                                   slow_start_config.value().slow_start_window()))
+                             : std::chrono::milliseconds(0)),
+      aggression_runtime_(
```

Comment:
This is no longer runtime and is a config value? Can you update this variable name? Also, you have error logging below which I think can also be caught now at config load time and/or PGV validation so that can be removed?

Reply:
it is a runtime double

Reply:
I see OK. I guess that's why we cant' have PGV, OK got it.